anndata/tests/test_concatenate.py

```diff
@@ -1433,3 +1433,20 @@ def test_outer_concat_outputs_nullable_bool_writable(tmp_path):
 
     adatas = concat({"a": a, "b": b}, join="outer", label="group")
     adatas.write(tmp_path / "test.h5ad")
+
+
+def test_concat_duplicated_columns(join_type):
+    # https://github.com/scverse/anndata/issues/483#issuecomment-1610579577
```

Comment:
that links to the “stale” comment, did you mean this?

```suggestion
    # https://github.com/scverse/anndata/issues/483#issuecomment-749321435
```

Reply:
I think I actually meant this

```suggestion
    # https://github.com/scverse/anndata/issues/483
```